iscript/tox.ini

```diff
@@ -8,9 +8,8 @@ usedevelop = false
 depends =
 skip_install = true
 commands =
-# issue with plistlib not having readPList: https://docs.python.org/3.9/whatsnew/3.9.html#removed
-#    docker build --build-arg PYTHON_VERSION=3.9.7 --build-arg PYTHON_REQ_SUFFIX=.py39 -t iscript-{envname}-py39-test -f Dockerfile.test .
-#    docker run --rm -v {toxinidir}/..:/app -v iscript-{envname}-py39-tox:/app/.tox -w /app/iscript iscript-{envname}-py39-test py39
+    docker build --build-arg PYTHON_VERSION=3.9.7 --build-arg PYTHON_REQ_SUFFIX=.py39 -t iscript-{envname}-py39-test -f Dockerfile.test .
+    docker run --rm -v {toxinidir}/..:/app -v iscript-{envname}-py39-tox:/app/.tox -w /app/iscript iscript-{envname}-py39-test py39
```

Comment:
I don't think we have been using docker tox testing, but if this works, thanks for figuring it out!